WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacyBannerViewController.swift

```diff
@@ -29,8 +29,11 @@ final class PrivacyBannerViewController: UIHostingController<PrivacyBanner> {
         // Make the banner scrollable when the banner height is bigger than the screen height.
         // Send it in the next run loop to avoid a recursive `viewDidLayoutSubviews`.
         bannerIntrinsicHeight = view.intrinsicContentSize.height
-        DispatchQueue.main.async {
-            self.rootView.shouldScroll = self.bannerIntrinsicHeight > self.view.frame.height
+        let shouldScroll = self.bannerIntrinsicHeight > self.view.frame.height
+        if self.rootView.shouldScroll != shouldScroll {
+            DispatchQueue.main.async {
+                self.rootView.shouldScroll = shouldScroll
+            }
```

Comment:
Added this check if the `should` scroll changed because CPU was running around 100%, this was running in loop.